### PR TITLE
updated badges

### PR DESCRIPTION
**Pull request by jspaaks**
- [x] code repository
- [x] license
- [x] community registry
- [x] citation
- [x] checklist

### CODE REVIEW

**Comment by jspaaks** [comment]
All badges updated except PyPI (pending #13 )

File: README.rst [hunk 1]
```diff
@@ -18,26 +18,26 @@
 (Customize these badges with your own links, and check https://shields.io/ or https://badgen.net/ to see which other badges are available.)
 
 .. |GitHub Badge| image:: https://img.shields.io/badge/github-repo-000.svg?logo=github&labelColor=gray&color=blue
-   :target: https://github.com/citation-file-format/cff-converter-python
+   :target: https://github.com/matchms/matchms
    :alt: GitHub Badge
 
 .. |License Badge| image:: https://img.shields.io/github/license/citation-file-format/cff-converter-python
```
Comment (fdiblen) [inline README.rst:24]: this is a badge for cff-converter

File: README.rst [hunk 2]
```diff
@@ -18,26 +18,26 @@
...
-   :target: https://github.com/citation-file-format/cff-converter-python
+   :target: https://github.com/matchms/matchms
    :alt: License Badge
 
 .. |PyPI Badge| image:: https://img.shields.io/pypi/v/cffconvert.svg?colorB=blue
```
Comment (fdiblen) [inline README.rst:28]: cff-corverter again

File: README.rst [hunk 3]
```diff
@@ -18,26 +18,26 @@
...
-.. |Research Software Directory Badge| image:: https://img.shields.io/badge/rsd-cffconvert-00a3e3.svg
-   :target: https://www.research-software.nl/software/cff-converter-python
+.. |Research Software Directory Badge| image:: https://img.shields.io/badge/rsd-matchms-00a3e3.svg
+   :target: https://www.research-software.nl/software/matchms
    :alt: Research Software Directory Badge
 
-.. |Zenodo Badge| image:: https://zenodo.org/badge/DOI/10.5281/zenodo.1162057.svg
-   :target: https://doi.org/10.5281/zenodo.1162057
+.. |Zenodo Badge| image:: https://zenodo.org/badge/DOI/10.5281/zenodo.3716379.svg
+   :target: https://doi.org/10.5281/zenodo.3716379
```
Comment (fdiblen) [inline README.rst:36]: Zenodo  title says Spec2Vec but github repo etc are all pointing to matchms

**Review by fdiblen — CHANGES_REQUESTED** [review]
Some changes needed for the badges and Zenodo

**Pull request merged**